src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAO.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.implicitConversions
 
 class GraphWorkspaceDAO extends WorkspaceDAO with GraphDAO {
 
-  override def save(workspace: Workspace, txn: RawlsTransaction): Workspace = txn withGraph { db =>
+  override def save(workspace: Workspace, txn: RawlsTransaction): WorkspaceContext = txn withGraph { db =>
```

Comment:
Also, the reason we have traits for DAOs is we could have different implementations - we don't now, but making this return WorkspaceContext makes this DAO inherently graph and google bucket